src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -326,6 +306,7 @@ public boolean isField(Element e) {
         return e.getKind() == FIELD;
     }
 
+    // Note that e.getKind().isInterface() is not the same as e.getKind() == INTERFACE
     public boolean isInterface(Element e) {
         return e.getKind() == INTERFACE;
```

Comment:
This seems asymmetrically different from `isClass`. We should come up with more consistent terminology/usage.

Reply:
isClassExact? isInterfaceExact?